src/main/java/javaserver/Response.java

```diff
@@ -1,6 +1,7 @@
 package javaserver;
 
 public class Response {
+
 	private String statusLine;
 	private String header;
 	private String body;
```

Comment:
As an aside, if you do happen to use the Builder pattern here. You may need to consider changing your `ResponseBuilder` object to something like `Responder` as this can get confusing!